reports/templegold-report.md

```diff
@@ -6783,12 +6783,6 @@ Use descriptive reason strings or custom errors for revert paths.
 	        require(address(pools[revalToken]) == address(0));
 	```
 
-- Found in contracts/util/ABDKMath64x64.sol [Line: 36](../tests/2024-07-templegold/protocol/contracts/util/ABDKMath64x64.sol#L36)
```

Comment:
This is blocked until we understand why this is happening @TilakMaddy

Reply:
Could this be hash collisions of sorts? In the workspace context, we have things like `assignments_context`, `returns_context`, etc... which are hashmaps from keys being the hash of the node and values being `NodeContext` ? 

This way, at the time of capturing, due to hash collision, we get the wrong value, here source unit id ??

Not sure, just speculation